cmd/nginx/flags.go

```diff
@@ -161,6 +160,7 @@ Feature backed by OpenResty Lua libraries. Requires that OCSP stapling is not en
 	)
 
 	flags.MarkDeprecated("sort-backends", "Feature removed because of the lua load balancer that removed the need of reloads for change in endpoints")
+	flags.MarkDeprecated("status-port", `Status port is a unix socket.`)
```

Comment:
This also applies to `sort-backends`, but why are we marking these flags deprecated when we are  removing them? Aren't they either deprecated or removed, but not both?

Reply:
fixed.